flixel/util/FlxCollision.hx

```diff
@@ -18,6 +18,7 @@ import flixel.tile.FlxTileblock;
  * @link http://www.photonstorm.com
  * @author Richard Davey / Photon Storm
 */
+@:access(flixel.FlxSprite)
```

Comment:
Hm, not a fan of this... Have you checked whether it makes a noticable performance difference to call `drawFrame()`? Thinking about it some more, it might actually not matter much, since it's a non-op if the sprite isn't `dirty`.